client/src/components/Questions/styles/questions.css

```diff
@@ -1,17 +1,13 @@
-#questions-widget {
-  margin: auto;
-  margin-bottom: 5em;
-  max-width: 1000px;
-}
+/* GENERAL TAGS */
 
-.qa.question {
-  display: flex;
-  align-items: baseline;
-  justify-content: space-between;
+div#questions-widget {
```

Comment:
I don't recommend using the div tag here. The id tag is as specific as it gets and wont overwrite anything else.